js/index.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function() {
+
+	$('form').submit(addCity);
+	
+
+	function addCity (event) { 
+		event.preventDefault();
+		var city = $('#city-type').val();
+
+		if (city === 'austin' || city === 'ATX' || city === 'Austin') { 
+			$('body').addClass('austin');
+			console.log('Austin!');
+			$('form').trigger("reset");
```

Comment:
This is awesome; I wouldn't have even thought to do it this way. Another way you could reset the form is by _setting_ the value to an empty string rather than _getting_ it, using jQuery's `.val()` function. That might look something like this:

```javascript
// set the value of the input with the id city-type to an empty string
$('#city-type').val('');
```